### PR TITLE
Update lizard docs

### PR DESCRIPTION
**Pull request by WolfSchouten**
Played around with the autodoc feature, seems to be working fine.

Please check it out.

### CODE REVIEW

File: source/d_geoblocks.rst [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 =========
 
 Digital geo services rely on high resolution big data coming from multiple real-time sources. 
-We developed GeoBlocks to make it more easy to process and analyse (real-time) that data to create a digital geo service. 
+We developed GeoBlocks to make it easier to process and analyse (real-time) that data to create a digital geo service. 
 
-GeoBlocks is a GIS modelling toolbox that allows you to make on-the-fly analysis using raster & vector data. 
-There are two basetype Blocks: `RasterBlocks <https://dask-geomodeling.readthedocs.io/en/latest/raster.html>`_ and `Geometry and Series Blocks <https://dask-geomodeling.readthedocs.io/en/latest/geometry.html#>`_. 
-A combination of blocks is called a "graph".
+GeoBlocks is a GIS modelling toolbox that allows you to make on-the-fly analysis using raster & vector data stored in Lizard.
```
Comment (caspervdw) [inline source/d_geoblocks.rst:8]:
```suggestion
GeoBlocks is a GIS modelling toolbox that allows you to make on-the-fly analyses using raster & vector data stored in Lizard. 
```

File: source/d_geoblocks.rst [hunk 2]
```diff
@@ -22,54 +22,24 @@ Open Source
...
-Individual labels (e.g. label linked to one building or parcel) can be found on the `labels endpoint <demo.lizard.net/api/v3/labels>`_.  
-Labels can be computed on the fly using the compute endpoint or a-sync using the Lizard Task Server. 
+GeoBlocks are configured in `JSON <https://en.wikipedia.org/wiki/JSON>`_. Your graph can be edited by POSTing or PATCHing to the Lizard API.
+For this we advise using `Postman <https://www.getpostman.com/>`_, but any application with an API component can be used. 
+Methods for POSTing geoblocks to Lizard differ between `RasterBlocks <https://demo.lizard.net/doc/rasterblocks.html#>`_ and `Geometry and Series Blocks <https://demo.lizard.net/doc//geometryblocks.html#>`_.
+On these pages explanation is provided on how to create a new GeoBlock in Lizard.
 
 Graphs
 ------
+A graph is a combination of multiple single Blocks which work together to create a "view" of one or more datasources in Lizard.
```
Comment (caspervdw) [inline source/d_geoblocks.rst:41]:
```suggestion
A graph is a combination of multiple single Blocks that work together to create a "view" of one or more datasources in Lizard.
```

File: source/d_geometryblocks.rst [hunk 3]
```diff
@@ -0,0 +1,190 @@
+.. _GeometryBlocksAnchor:
+
+==========================
+Geometry and Series Blocks
+==========================
+
+GeometryBlocks are the GeoBlocks which assimilate and operate rasterdata into vector data.
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:7]:
```suggestion
GeometryBlocks are the GeoBlocks that modify geometries or combine rasters with geometries.
```

File: source/d_geometryblocks.rst [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+They can sample rasters and perform geometric operations like intersections and geometric differences.
+The output from a Geometry GeoBlock is a label value.
+This means that the operation should always end with a classification of one of the feature columns into a label value. 
+The result of the GeometryBlock can be requested through the API.
+Such request returns the label for a geometric feature along with a defined list of intermediate results.
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:12]:
```suggestion
Such request returns the label for a geometric feature along with a predefined list of intermediate results.
```

File: source/d_geometryblocks.rst [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+Creating a new GeometryBlock
+----------------------------
+
+To create a new GeometryBlock you have to POST the graph directly to the labeltypes api endpoint: https://demo.lizard.net/api/v3/labeltypes/.
+Unlike for RasterBlocks it is not possible to define a GeometryBlock without the API.
+Once the graph has been posted it is possible to PATCH changes and alter the structure of the graph.
+If you want to patch changes to the graph this can be done by providing a valid JSON object for its source element,
+and perform a patch on https://demo.lizard.net/api/v3/labeltypes/{uuid of the new labeltype}/.
+
+Currently it is not possible to visualize the resultant labels in the Lizard portals.
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:23]:
```suggestion
Currently it is not possible to visualize the resulting labels in the Lizard portals.
```

File: source/d_geometryblocks.rst [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+https://demo.lizard.net/api/v3/labeltypes/{label type uuid}/compute/?geom_intersects=POINT(4.46648 51.92938).
+It is also possible to pre-compute larger amounts of labels. By doing so it becomes possible to quickly request multiple
+labels or label statistics. 
+
+To pre-compute labels for a specific region you have to send a POST request on the labeltype endpoint, for example:
+https://demo.lizard.net/api/v3/labeltypes/{label type uuid}/compute/?boundary_id=95246&start=2018-10-01T01:00:00Z&tile_size=500&tile_projection=EPSG:28992&mode=centroid.
+
+In the pre-compute you have to provide some sort of geographic bounding area.
+This can be through a predefined boundary in Lizard (like in the example) or a geometric bounding box.
+The start parameter applies a temporal filter to the features included.
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:34]:
```suggestion
The start parameter applies a temporal filter to the included features.
```

File: source/d_geometryblocks.rst [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+Features which are valid at the start date are computed, features which were built after or removed before the start-date are not computed.
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:35]:
```suggestion
Only features that are valid at the start date are computed; features that were created after or removed before the start-date are not included in the computation. 
```

File: source/d_geometryblocks.rst [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+
+In the pre-compute you have to provide some sort of geographic bounding area.
+This can be through a predefined boundary in Lizard (like in the example) or a geometric bounding box.
+The start parameter applies a temporal filter to the features included.
+Features which are valid at the start date are computed, features which were built after or removed before the start-date are not computed. 
+
+Label endpoint
+--------------
+
+Labels which have been pre-computed are stored in the labels endpoint of the API: https://demo.lizard.net/api/v3/labels/.
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:40]:
```suggestion
Labels that have been pre-computed are stored in the labels endpoint of the API: https://demo.lizard.net/api/v3/labels/.
```

File: source/d_geometryblocks.rst [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+By using this endpoint it is possible to request both individual labels and label statistics. Through a GET request it is possible to determine statistics of the entire labeltype or specific regions: https://demo.lizard.net/api/v3/labels/counts/?label_type__uuid={label type uuid}. 
+
+Labels which have been pre-computed are stored in the labels endpoint of the API: https://demo.lizard.net/api/v3/labels/.
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:43]: Duplicate paragraph

File: source/d_geometryblocks.rst [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+                    "buildings",
+                    "label_value",
+                    "label"
+                ]
+            },
+            "name": "result"
+        }
+    }
+
+Geometry output
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:101]: This paragraph contains info that you already included above. Can be omitted

File: source/d_geometryblocks.rst [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+
+:mod:`dask_geomodeling.geometry.geom_operations`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: dask_geomodeling.geometry.geom_operations
+   :members:
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.parallelize`
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:153]: This is not allowed through the API, should be omitted

File: source/d_geometryblocks.rst [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+
+:mod:`dask_geomodeling.geometry.set_operations`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: dask_geomodeling.geometry.set_operations
+   :members:
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.sources`
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:169]: Should be omitted as well

File: source/d_geometryblocks.rst [hunk 13]
```diff
@@ -0,0 +1,190 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: dask_geomodeling.geometry.sources
+   :members:
+   :exclude-members: get_sources_and_requests, process
+
+
+:mod:`dask_geomodeling.geometry.sinks`
```
Comment (caspervdw) [inline source/d_geometryblocks.rst:177]: This as well

File: source/d_rasterblocks.rst [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+============
+RasterBlocks
+============
+
+RasterBlocks are the GeoBlocks that work with rasterdata. RasterBlocks operate on `Lizard Rasterdata <https://docs.lizard.net/c_datatypes.html#rasters>`_.
+
+Creating a new RasterBlock
+==========================
+
+The first step of creating a new raster GeoBlock is making a new Lizard raster instance which will contain your graph.
```
Comment (caspervdw) [inline source/d_rasterblocks.rst:12]:
```suggestion
The first step of creating a new raster GeoBlock is making a new Lizard raster instance that will contain your graph.
```

File: source/d_rasterblocks.rst [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+- Search for your raster in the catalogue.
+- Open the raster in the API using the Open in API button. 
+- Follow the link mentioned in the source_url attribute. 
+
+.. image:: /images/d_geoblocks_01.png
+
+Operations
+==========
+
+Wolf, work your magic
```
Comment (caspervdw) [inline source/d_rasterblocks.rst:62]: Todo ;)

File: source/d_geometryblocks.rst (hunk 9 above)
Comment (WolfSchouten) [reply source/d_geometryblocks.rst:43]: removed the duplicate

File: source/d_geometryblocks.rst (hunk 11 above)
Comment (WolfSchouten) [reply source/d_geometryblocks.rst:153]: You mean the 'parralelize'?

File: source/d_geometryblocks.rst (hunk 10 above)
Comment (WolfSchouten) [reply source/d_geometryblocks.rst:101]: Leaving it in for now, the actual text was written by Martijn Krol. Going to discuss with him how to better communicate these options.

File: source/d_geometryblocks.rst (hunk 11 above)
Comment (caspervdw) [reply source/d_geometryblocks.rst:153]: Yes, parallelize, sources and sinks are internal and should not be configured through the API

File: source/d_rasterblocks.rst (hunk 15 above)
Comment (WolfSchouten) [reply source/d_rasterblocks.rst:62]: Missed that one, fixing now :)

File: source/d_geometryblocks.rst (hunk 11 above)
Comment (WolfSchouten) [reply source/d_geometryblocks.rst]: Ok, removed!

**Comment by WolfSchouten** [comment]
Made all requested changes, except for one.

**Pull request merged**